node/pkg/governor/governor.go

```diff
@@ -110,8 +112,10 @@ type (
 	// `transfers` with positive Value represent outgoing transfers from the emitterChainId. Transfers with negative
 	// Value represent incoming transfers of Assets that can Flow Cancel.
 	chainEntry struct {
-		emitterChainId          vaa.ChainID
-		emitterAddr             vaa.Address
+		emitterChainId vaa.ChainID
+		emitterAddr    vaa.Address
+		// The amount of notional value that can leave the chain during the period
+		// defined by the sliding window.
 		dailyLimit              uint64
```

Comment:
Makes sense to change this variable name too imo

Reply:
Ahhh ok yeah if Bruce and co don't think it's worth making a breaking change here then I agree you can revert back to the commit pre this change

Reply:
I think it's reasonable to change it if you want. Anyone who uses it will just need to make a change whenever they decide to pick up a new version of the package. 🤷